dev-libs/libstrophe/libstrophe-0.9.2.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A simple, lightweight C library for writing XMPP clients"
+HOMEPAGE="http://strophe.im/libstrophe"
+SRC_URI="https://github.com/strophe/libstrophe/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-3 MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="libxml2 +tls doc"
+
+DEPEND="dev-libs/openssl:0=
+	!libxml2? ( dev-libs/expat )
+	libxml2? ( dev-libs/libxml2 )"
```

Comment:
This logic is good, but I would personally switch the USE flag from libxml2 to expat and switch this around. 

Whatever you decide to do, add a slot operator to libxml2, `dev-libs/libxml2:2`.